notification-portlet-webapp/src/main/resources/META-INF/persistence.xml

```diff
@@ -54,6 +59,9 @@
                  hsqldb we need this to be update. Users *should* do a schema update before switching to new versions, or
                  this should hopefully not do something negative if it ever does update the schema (typically doesn't). -->
             <property name="hibernate.hbm2ddl.auto" value="update" />
+            
+            <!-- Dialect needed for POSTGRES to work -->
+            <property name="hibernate.dialect" value="${environment.build.hibernate.dialect}"/>
```

Comment:
The variable environment.build.hibernate.dialect is only available in the uportal build process.  In the portlet it should default to hsql.  An overlay persistence.xml should be added to uportal-portlets-overlay/NotificationPortlet that has $environment.build.hibernate.dialect, and the overlay's NotificationPortlet/pom.xml adjusted if need be to do filtering on persistence.xml.